src/flow-hash.c

```diff
@@ -665,6 +674,7 @@ static Flow *FlowGetNew(ThreadVars *tv, FlowLookupStruct *fls, Packet *p)
     if (g_eps_flow_memcap != UINT64_MAX && g_eps_flow_memcap == p->pcap_cnt) {
         NoFlowHandleIPS(p);
         StatsIncr(tv, fls->dtv->counter_flow_memcap);
+        FlowExceptionPolicyStatsIncr(tv, fls, flow_config.memcap_policy);
```

Comment:
why is this call not next to a expection policy apply function?

Reply:
I guess I just thought to call the incr in the same place that the other stats were increased 🤔  will change.